specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/secrets.json

```diff
@@ -483,5 +476,19 @@
         "user_impersonation": "impersonate your user account"
       }
     }
+  },
+  "responses": {
```

Comment:
Don't redefine these. Use the ones from the common include like I did in the data-plane. Or are all the mgmt-plane swaggers standalone?

Reply:
I think some of the definitions can move to a new common.json file, which we don't have in previous version in resource manager. I can start with the error message and this preview version. Any concerns?